### ISSUE TITLE
Compatibility issue with Angular version 8.2

### ISSUE THREAD

**Issue by meenalgupta23** [guillotinaweb/ngx-schema-form#466]

**Comment by ebrehault** [comment]
If you mean ngx-schema-form 2.8 is not compatible with Angular 15, you can use ngx-schema-form 2.9.0 which has just been released.

**Comment by meenalgupta23** [comment]
No, I am working on Angular version 2.8.14 and it is not compatible with any of the released ngx-schema-form version.

**Comment by ebrehault** [comment]
Oh waooh, that's a super old version :)

Well, it is not published on npmjs, but you can check out tag `1.0.0-beta.6` of this repository, that's the last version before we have upgraded to Angular 4.
At the time the package was named `angular2-schema-form`. There will be probably few bugs and it will not offer all the features we have now, but it should work.

**Comment by meenalgupta23** [comment]
Oops! I'm so sorry, there was a little typo, its actually angular 8.2.14

**Comment by ebrehault** [comment]
Then use version 2.4.4 (it's on npm)
